apps/nextjs-example/components/transactionFlow/MultiAgent.tsx

```diff
@@ -43,7 +43,7 @@ export default function MultiAgentTransaction({
       throw new Error("no network");
     }
 
-    const secondarySigner = Account.generate();
+    const secondarySigner = Ed25519Account.generate();
```

Comment:
nit:

dont really need this change as `Account.generate()` already creates a legacy Ed25519Account

Reply:
Ah ok, I only did it to make it more explicit :p, wasn't sure if it'd type it correctly